nemo/collections/nlp/models/text_classification/text_classification_model.py

```diff
@@ -314,3 +315,37 @@ def list_available_models(cls) -> Optional[Dict[str, str]]:
     @classmethod
     def from_pretrained(cls, name: str):
         pass
+
+    @classmethod
+    def _prepare_for_export(self):
+        return self.bert_model._prepare_for_export()
+
+    @classmethod
+    def export(
+        self,
+        output: str,
+        input_example=None,
+        output_example=None,
+        verbose=False,
+        export_params=True,
+        do_constant_folding=True,
+        keep_initializers_as_inputs=False,
+        onnx_opset_version: int = 12,
+        try_script: bool = False,
+        set_eval: bool = True,
+        check_trace: bool = True,
+        use_dynamic_axes: bool = True,
+    ):
+        return self.bert_model.export(
+            output,
```

Comment:
Can we move these to ModelNLP @ericharper is adding in his PR? https://github.com/NVIDIA/NeMo/pull/1226

Reply:
@ekmb not sure I understood completely

Reply:
@drnikolaev  - my bad, I should've noticed this earlier. But "main" is going to be released on friday, before then all non-bugfixes have to go to "dev".

Reply:
moved to dev under ModelNLP